test/ivToken/IvToken.t.sol

```diff
@@ -0,0 +1,157 @@
+
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.16;
+
+import "forge-std/Test.sol";
+import "../TestCommons.sol";
+import "../../contracts/tokens/IvToken.sol";
+import "../../contracts/mocks/tokens/DaiMockedToken.sol";
+
+contract IvTokenTest is Test, TestCommons {
+    IvToken internal _ivToken;
+    DaiMockedToken internal _daiMockedToken;
+    address internal _admin;
+    address internal _userOne;
+    address internal _userTwo;
+
+    function setUp() public {
+        _ivToken = new IvToken("IvToken", "IVT", address(0x6B175474E89094C44Da98b954EedeAC495271d0F));
```

Comment:
Please add a description to this address `0x6B175474E89094C44Da98b954EedeAC495271d0F` (random or so)